c2corg_ui/templates/waypoint/edit.html

```diff
@@ -35,6 +35,7 @@ <h1 ng-init="id = ${waypoint_id if waypoint_id else 0}" class="text-center">
     <span class="ng-hide">
       {{editCtrl.updateMaxSteps(waypoint.waypoint_type)}} 
       {{editCtrl.waypoint_type = waypoint.waypoint_type}}
+      {{document = waypoint}}
```

Comment:
Does this mean this hidden span is used to set vars?
Isn't it a bit of a hack?
Can't you an ``ng-init`` for instance?